2024/survey.yaml

```diff
@@ -198,6 +198,12 @@ questions:
       - Embedded/IoT devices
   - prompt: Which version of Nix do you use? (only write the SemVer, e.g. "2.18.11", get it with `nix-env --version`)
     type: text
+  - prompt: What do you care about most for Nix?
+    type: single
+    choices:
+      - Stability
+      - Performance
+      - New features
```

Comment:
What do you mean by "new features"?

Reply:
Getting new things added to Nix.

The goal of the question (if that makes it clearer) is for Nix maintainers to get an idea of what to prioritize between fixing bugs/increasing the reliability of the code base, making Nix faster, and bringing in the bazillion of exciting potential new developments that allow Nix to do more stuff.

Reply:
> bringing in the bazillion of exciting potential new developments that allow Nix to do more stuff

For example?

Reply:
I can give some concrete big examples (lazy trees, dynamic derivations, Windows support, new version of the Nix language with less corner cases, etc..), but I don't think they would be relevant because the bulk of this is really the many small improvements from https://github.com/NixOS/nix/labels/feature which don't follow any particular axis

Reply:
I think it's important to make each proposed choice more understandable. As it is the question and answers are too vague to lead to an actionnable analysis.